frappe/email/inbox.py

```diff
@@ -145,7 +145,7 @@ def make_lead_from_communication(communication, ignore_communication_links=False
 	lead_name = None
 	if doc.sender:
 		lead_name = frappe.db.get_value("Lead", {"email_id": doc.sender})
```

Comment:
Why does this exist in Frappe?

Reply:
Will fix this

Reply:
Even Rushabh has told to Anurag in the PR  https://github.com/frappe/frappe/pull/7248 but still he merged